app/lib/cds_importer/entity_mapper/base_mapper.rb

```diff
@@ -59,27 +82,14 @@ def before_building_model(&block)
         end
       end
 
-      NATIONAL = 'N'.freeze
-      APPROVED_FLAG = '1'.freeze
-      STOPPED_FLAG = '1'.freeze
-      PATH_SEPARATOR = '.'.freeze
-      METAINFO = 'metainfo'.freeze
-      BASE_MAPPING = {
-        'validityStartDate' => :validity_start_date,
-        'validityEndDate' => :validity_end_date,
-        'metainfo.origin' => :national,
-        'metainfo.opType' => :operation,
-        'metainfo.transactionDate' => :operation_date,
-      }.freeze
-
-      def initialize(values)
-        @values = values.slice(*mapping_with_key_as_array.keys.map { |k| k[0] }.uniq)
```

Comment:
This side effect in the constructor literally adds no value in terms of the built entity that results from calling the `#parse`.

I typically really don't like side effects in the constructor because it's hard to test the result of the side effect on the instance/collaborator. In this case the side effect was noop/does nothing